### PR TITLE
PLATEXP-1084: User Management - Getting started with Role-Based Access Control (RBAC)

### PR DESCRIPTION
**Pull request by asmita32675**
https://rackspace.atlassian.net/browse/PLATEXP-1084

### CODE REVIEW

**Review by Rohi2305 — CHANGES_REQUESTED** [review]

File: content/how-to/account-management/overview-role-based-access-control-rbac/index.md [hunk 1]
```diff
@@ -5,185 +5,126 @@ title: Learn about Role-Based Access Control (RBAC)
...
+### Implementation of RBAC
+RBAC gives customers a greater degree of control over cloud resource use, with an additional layer of system security.
 
+### Types of Users
 RBAC has the following types of users:
 
--   **Account owner** - The account owner is the primary contact for the
-    account and has full permissions to execute all capabilities for
-    every product available. Each account has a single account owner.
+-   **Account owner** - The account owner, who serves as the main point of contact for the account, is fully authorized to use all available product features. There is just one account owner per account.
```
[the remark this answers is not in the document]
Comment (asmita32675) [reply content/how-to/account-management/overview-role-based-access-control-rbac/index.md:32]: Updated

File: content/how-to/account-management/overview-role-based-access-control-rbac/index.md [hunk 2]
```diff
@@ -5,185 +5,126 @@ title: Learn about Role-Based Access Control (RBAC)
...
 
--   **Account user** - The account user is a user who has been added by
-    the account owner and has been assigned to specific product or
-    account roles.
-
-### What actions are restricted to the account owner?
+-   **Account user** - The account user is a user that the account owner added and who has been given access to particular products or account roles.
```
[the remark this answers is not in the document]
Comment (asmita32675) [reply content/how-to/account-management/overview-role-based-access-control-rbac/index.md:34]: Done

**Review by Rohi2305 — CHANGES_REQUESTED** [review]

File: content/how-to/account-management/account-management-faq/index.md [hunk 3]
```diff
@@ -5,17 +5,17 @@ title: Account Management FAQ
...
 ---
 
-### Access control
+### Access Control
 
-{{< accordion title="What is the cost to opt in to RBAC?" col="in" href="accordion16" >}}
+{{< accordion title="What would it cost to choose RBAC?" col="in" href="accordion16" >}}
 
-RBAC, a no-cost feature, is available to selected public cloud services with an API.
+RBAC is a free feature that is accessible through an API for some public cloud providers.
```
[the remark this answers is not in the document]
Comment (asmita32675) [reply content/how-to/account-management/account-management-faq/index.md:18]: Updated.

File: content/how-to/account-management/account-management-faq/index.md [hunk 4]
```diff
@@ -30,13 +30,12 @@ Yes, RBAC is available to all Rackspace customers.
 
 {{< accordion title="When I add a user login to an existing contact, can I configure custom role assignments?" col="in" href="accordion19" >}}
 
-Yes, you can. For instructions, see [Use Role-Based Access Control
-(RBAC)](/support/how-to/managing-role-based-access-control-rbac).
+Yes, you can. For instructions, see [Use Role-Based Access Control (RBAC)](/support/how-to/use-role-based-access-control-rbac).
 {{</ accordion >}}
 
-{{< accordion title="How do I query the capability of a given user?" col="in" href="accordion20" >}}
+{{< accordion title="How can I find out a user's capabilities?" col="in" href="accordion20" >}}
```
[the remark this answers is not in the document]
Comment (asmita32675) [reply content/how-to/account-management/account-management-faq/index.md:36]: Updated

File: content/how-to/account-management/account-management-faq/index.md [hunk 5]
```diff
@@ -30,13 +30,12 @@ Yes, RBAC is available to all Rackspace customers.
...
 
-You can query the roles for a given user by using the [List global roles assigned to a user](https://docs.rackspace.com/docs/cloud-identity/v2/api-reference/role-operations/#list-global-roles-assigned-to-a-user) API operation described in the *Identity API Guide*. Additionally, you can view the roles that a user has through the [Cloud Control Panel](https://login.rackspace.com).
+Using the [List global roles assigned to a user](https://docs.rackspace.com/docs/cloud-identity/v2/api-reference/role-operations/#list-global-roles-assigned-to-a-user) API methods defined in the *Identity API Guide*. you can check about the roles for a certain user. Through the [Cloud Control Panel](https://login.rackspace.com), you can also see the roles that each user has.
 {{</ accordion >}}
 
 {{< accordion title="Can I view multiple accounts in the Cloud Control Panel?" col="in" href="accordion21" >}}
```
[the remark this answers is not in the document]
Comment (asmita32675) [reply content/how-to/account-management/account-management-faq/index.md:41]: Updated

File: content/how-to/account-management/account-management-faq/index.md [hunk 6]
```diff
@@ -51,27 +50,22 @@ Per-billing and per-user usage are not offered at this time.
...
-All users can create support tickets. However, only the account owner is
-updated on the ticket status.
+All users can create support tickets. However, only the account owner is updated on the ticket status.
 {{</ accordion >}}
 
 {{< accordion title="Can I delegate all functionality as an account owner?" col="in" href="accordion23" >}}
 
-Not at this time, but we are working on developing and delivering that
-functionality.
+Not at this time, but we are working on developing and delivering that functionality.
```
[the remark this answers is not in the document]
Comment (asmita32675) [reply content/how-to/account-management/account-management-faq/index.md:58]: Updated as Yes, now account owner can delegate all functionality.

File: content/how-to/account-management/account-management-faq/index.md [hunk 7]
```diff
@@ -51,27 +50,22 @@ Per-billing and per-user usage are not offered at this time.
...
 {{</ accordion >}}
 
 {{< accordion title="Does RBAC provide fine-grained access control for specific servers, directories, or files?" col="in" href="accordion24" >}}
 
-RBAC is limited to granting a user access to a given product. RBAC is
-not granular enough to restrict access to individual servers, files, or
-directories.
+RBAC is limited to granting a user access to a given product. RBAC is not granular enough to restrict access to individual servers, files, or directories.
```
[the remark this answers is not in the document]
Comment (asmita32675) [reply content/how-to/account-management/account-management-faq/index.md:63]: Done

File: content/how-to/account-management/account-management-faq/index.md [hunk 8]
```diff
@@ -51,27 +50,22 @@ Per-billing and per-user usage are not offered at this time.
...
 {{</ accordion >}}
 
 {{< accordion title="Does the API offer more features for RBAC than the Cloud Control Panel offers?" col="in" href="accordion25" >}}
 
-Yes, at this time more features are available through the API. For more
-information about using the API for RBAC, see the [Rackspace API Documentation](https://docs.rackspace.com/docs/).
+Yes, at this time more features are available through the API. For more information about using the API for RBAC, see the [Rackspace API Documentation](https://docs.rackspace.com/docs/).
 {{</ accordion >}}
 
 {{< accordion title="Do account users automatically inherit the RackConnect and Managed Cloud features?" col="in" href="accordion26" >}}
```
[the remark this answers is not in the document]
Comment (asmita32675) [reply content/how-to/account-management/account-management-faq/index.md:71]: Done

File: content/how-to/account-management/account-management-faq/index.md [hunk 9]
```diff
@@ -81,10 +75,7 @@ Yes.
 
 {{< accordion title="How does RBAC work with MyRackspace?" col="in" href="accordion27" >}}
 
-With RBAC, account owners can leave permissions as
-they are in MyRackspace or set up restricted access to specific products
-for users through the Cloud Control Panel. For more detailed information,
-see [Learn about Role-Based Access Control (RBAC)](/support/how-to/overview-role-based-access-control-rbac/).
+With RBAC, account owners can leave permissions as they are in MyRackspace or set up restricted access to specific products for users through the Cloud Control Panel. For more detailed information, see Learn about Role-Based Access Control (RBAC) for [Rackspace Cloud](/support/how-to/overview-role-based-access-control-rbac-cloud/) and [Rackspace Dedicated](/support/how-to/overview-role-based-access-control-rbac-dedicated/).
```
[the remark this answers is not in the document]
Comment (asmita32675) [reply content/how-to/account-management/account-management-faq/index.md:78]: Updated

File: content/how-to/account-management/account-management-faq/index.md [hunk 10]
```diff
@@ -95,10 +86,7 @@ permission scheme among dedicated, hybrid, and cloud customers in the future.
 
 {{< accordion title="Are servers, containers, files, databases, and load balancers shared, or do users each have their own set of resources?" col="in" href="accordion29" >}}
```
[the remark this answers is not in the document]
Comment (asmita32675) [reply content/how-to/account-management/account-management-faq/index.md:87]: Done

File: content/how-to/account-management/overview-role-based-access-control-rbac-cloud/index.md [hunk 11]
```diff
@@ -0,0 +1,146 @@
+---
+permalink: overview-role-based-access-control-rbac-cloud
+audit_date: '2018-02-16'
+title: Overview Role Based Access Control for Cloud
+type: article
+created_date: '2013-04-14'
+created_by: Renee Rendon
+last_modified_date: '2022-08-25'
```
[the remark this answers is not in the document]
Comment (asmita32675) [reply content/how-to/account-management/overview-role-based-access-control-rbac-cloud/index.md:8]: done

File: content/how-to/account-management/overview-role-based-access-control-rbac-cloud/index.md [hunk 12]
```diff
@@ -0,0 +1,146 @@
...
+last_modified_by: Asmita Nakwa
+product: Account Management
+product_url: account-management
+---
+
+**Previous Section:** [Getting started with Role-Based Access Control (RBAC)](/support/how-to/getting-started-with-role-based-access-control-rbac)
+
+This article answers basic questions about the Role-Based Access Control (RBAC) service.
```
[the remark this answers is not in the document]
Comment (asmita32675) [reply content/how-to/account-management/overview-role-based-access-control-rbac-cloud/index.md:16]: done

File: content/how-to/account-management/account-management-faq/index.md [hunk 13]
```diff
@@ -30,13 +30,12 @@ Yes, RBAC is available to all Rackspace customers.
 
 {{< accordion title="When I add a user login to an existing contact, can I configure custom role assignments?" col="in" href="accordion19" >}}
```
[the remark this answers is not in the document]
Comment (asmita32675) [reply content/how-to/account-management/account-management-faq/index.md:31]: Modified as "When I add a user login credential to an existing contact, can I configure custom role assignments?"

File: content/how-to/account-management/account-management-faq/index.md [hunk 14]
```diff
@@ -81,10 +75,7 @@ Yes.
...
 {{< accordion title="How does RBAC work with MyRackspace?" col="in" href="accordion27" >}}
 
-With RBAC, account owners can leave permissions as
-they are in MyRackspace or set up restricted access to specific products
-for users through the Cloud Control Panel. For more detailed information,
-see [Learn about Role-Based Access Control (RBAC)](/support/how-to/overview-role-based-access-control-rbac/).
+With RBAC, account owners can leave permissions as they are in MyRackspace or set up restricted access to specific products for users through the Cloud Control Panel. For more detailed information, see Learn about Role-Based Access Control (RBAC) for [Rackspace Cloud](/support/how-to/overview-role-based-access-control-rbac-cloud/) and [Rackspace Dedicated](/support/how-to/overview-role-based-access-control-rbac-dedicated/).
 {{</ accordion >}}
 
 {{< accordion title="Can I link dedicated, hybrid, and cloud permissions?" col="in" href="accordion28" >}}
```
[the remark this answers is not in the document]
Comment (asmita32675) [reply content/how-to/account-management/account-management-faq/index.md:81]: Modified the answer: "Currently it is not possible to have one user credential for all accounts. A Dedicated account requires one set of user credentials, and Cloud accounts require a different set of user credentials. We do provide an Account Linking feature that allows a Dedicated account to link Cloud Accounts, and you can manage the Dedicated users that are allowed to click and access a Cloud Account to perform either Administrative, Editing, or Viewing functions. See the following article to configure this feature."

File: content/how-to/account-management/account-management-faq/index.md [hunk 15]
```diff
@@ -5,17 +5,17 @@ title: Account Management FAQ
 type: article
 created_date: '2019-05-16'
 created_by: Stephanie Fillmon
-last_modified_date: '2020-09-08'
-last_modified_by: Stephanie Fillmon
+last_modified_date: '2020-09-14'
```
Comment (ssalinas24) [inline content/how-to/account-management/account-management-faq/index.md:8]: date can be updated once we are about to release

File: content/how-to/account-management/account-management-faq/index.md [hunk 16]
```diff
@@ -28,15 +28,14 @@ Customers can opt out by simply removing the account users.
...
-(RBAC)](/support/how-to/managing-role-based-access-control-rbac).
+Yes, you can. For instructions, see [Use Role-Based Access Control (RBAC)](/support/how-to/use-role-based-access-control-rbac).
 {{</ accordion >}}
 
-{{< accordion title="How do I query the capability of a given user?" col="in" href="accordion20" >}}
+{{< accordion title="How can I find out a user's roles and responsibilities?" col="in" href="accordion20" >}}
 
-You can query the roles for a given user by using the [List global roles assigned to a user](https://docs.rackspace.com/docs/cloud-identity/v2/api-reference/role-operations/#list-global-roles-assigned-to-a-user) API operation described in the *Identity API Guide*. Additionally, you can view the roles that a user has through the [Cloud Control Panel](https://login.rackspace.com).
+Using the [List global roles assigned to a user](https://docs.rackspace.com/docs/cloud-identity/v2/api-reference/role-operations/#list-global-roles-assigned-to-a-user) API methods defined in the *Identity API Guide*. you can check about the roles for a certain user. Through the [Cloud Control Panel](https://login.rackspace.com), you can also see the roles that each user has.
```
Comment (ssalinas24) [inline content/how-to/account-management/account-management-faq/index.md:38]: Keep the new links but revert to the original text and sentences that were around it. The new text is more confusing.

File: content/how-to/account-management/account-management-faq/index.md [hunk 17]
```diff
@@ -51,54 +50,43 @@ Per-billing and per-user usage are not offered at this time.
 
 {{< accordion title="Which users can create support tickets?" col="in" href="accordion22" >}}
 
-All users can create support tickets. However, only the account owner is
-updated on the ticket status.
+All users can create support tickets. However, only the account owner is updated on the ticket status.
```
Comment (ssalinas24) [inline content/how-to/account-management/account-management-faq/index.md:53]: The above would probably be incorrect. To be more precise the answer could be as follows:
`All users with account administration permissions are allowed to create support tickets related to account or device changes. Users limited to device permissions are only allowed to create tickets for the specific devices they are granted permissions on. For further information about Ticket Creation, please visit the Fabric Ticketing section.`

You can link over to the Fabric Ticketing Create Ticket section if possible.

File: content/how-to/account-management/account-management-faq/index.md [hunk 18]
```diff
@@ -51,54 +50,43 @@ Per-billing and per-user usage are not offered at this time.
...
 {{</ accordion >}}
 
 {{< accordion title="Can I delegate all functionality as an account owner?" col="in" href="accordion23" >}}
 
-Not at this time, but we are working on developing and delivering that
-functionality.
+Yes, now account owner can delegate all functionality.
```
Comment (ssalinas24) [inline content/how-to/account-management/account-management-faq/index.md:58]: remove the word `now`. update to be `Yes, an account owner can delegate all functionality.`

**Review by ssalinas24 — CHANGES_REQUESTED** [review]
Here is initial review from today. I will continue again this afternoon, and we can also continue during tomorrow's meeting if needed since this is a pretty big PR.

File: content/how-to/account-management/using-RBAC-with-MyRackspace/index.md [hunk 19]
```diff
@@ -0,0 +1,177 @@
...
+
+The account owner implements Role Based Access Control (RBAC) by adding users to the account and assigning roles. This article is intended to guide account owners through this process by using the [Rackspace Dedicated](https://login.rackspace.com/).
+
+For information about setting up RBAC through the API, see the [Rackspace Identity API Guide](https://docs.rackspace.com/docs/cloud-identity/v2/developer-guide/).
+
+**Note:** It is possible to assign a mix of multiple-product roles and per-product roles to one user through the API. The most permissive role determines the user's level of access.
+
+### Account Credentials
+
+Rackspace recommends that you can change the account password before adding new users to the account.
```
[the remark this answers is not in the document]
Comment (asmita32675) [reply content/how-to/account-management/using-RBAC-with-MyRackspace/index.md:24]: Done

File: content/how-to/account-management/use-role-based-access-control-rbac/index.md [hunk 20]
```diff
@@ -0,0 +1,98 @@
...
+When new users are created, a temporary password is assigned to them. They should change the temporary password at their first login.
+
+Also, new users receive an email from Rackspace notifying them that they have been added to the account. They receive information on how to sign up for an account in the email.
+
+### Create New Users
+
+To create a new user, use the following steps:
+
+1. Log in to the [Cloud Control Panel](https://login.rackspace.com/).
+<img width="508" alt="multi-account-RBAC-login" src="/support/how-to/use-role-based-access-control-rbac/multi-account-RBAC-login.png">
```
[the remark this answers is not in the document]
Comment (asmita32675) [reply content/how-to/account-management/use-role-based-access-control-rbac/index.md:35]: Updated

File: content/how-to/account-management/use-role-based-access-control-rbac/index.md [hunk 21]
```diff
@@ -0,0 +1,98 @@
...
+
+2. In the upper-right corner of the control panel, click **Account > User Management.**
+<img width="946" alt="multi_account_rbac1" src="/support/how-to/use-role-based-access-control-rbac/multi_account_rbac1.png">
```
[the remark this answers is not in the document]
Comment (asmita32675) [reply content/how-to/account-management/use-role-based-access-control-rbac/index.md:38]: Done

File: content/how-to/account-management/overview-role-based-access-control-rbac-cloud/index.md [hunk 22]
```diff
@@ -0,0 +1,161 @@
...
+
+<image width="919" alt="Select user to set the permission" src="/support/how-to/overview-role-based-access-control-rbac-cloud/rbac3.png">
+
+**Step 4.** Under the Rackspace Cloud** option in the **Product Permissions** section, click **Edit** next to the **PRODUCT ACCESS** tab.
+
+<image width="900" alt="Click Edit to edit product access" src="/support/how-to/overview-role-based-access-control-rbac-cloud/rbac4.png">
+
+The **Rackspace Cloud Permissions** window opens as shown below.
+
+<image width="203" alt="rbac5" src="/support/how-to/overview-role-based-access-control-rbac-cloud/rbac5.png">
```
Comment (ssalinas24) [inline content/how-to/account-management/overview-role-based-access-control-rbac-cloud/index.md:98]: alt tag still incorrect

File: content/how-to/account-management/overview-role-based-access-control-rbac-dedicated/index.md [hunk 23]
```diff
@@ -0,0 +1,137 @@
...
+- Assign By User
+- Assign By Product
+- Manage Groups
+- Global Permissions
+
+**Assign By User**
+
+**Step 1.** To assign permission **By User**, select a user/user group from the column. 
+
+<img width="931" alt="rbac8" src="/support/how-to/overview-role-based-access-control-rbac-dedicated/rbac8.png">
```
Comment (ssalinas24) [inline content/how-to/account-management/overview-role-based-access-control-rbac-dedicated/index.md:94]: alt tag still incorrect

File: content/how-to/account-management/overview-role-based-access-control-rbac-dedicated/index.md [hunk 24]
```diff
@@ -0,0 +1,137 @@
...
+
+**Step 2.** Select the checkbox under **Direct** permissions you want to assign to that user/user group.
+
+<img width="928" alt="rbac9" src="/support/how-to/overview-role-based-access-control-rbac-dedicated/rbac9.png">
```
Comment (ssalinas24) [inline content/how-to/account-management/overview-role-based-access-control-rbac-dedicated/index.md:98]: alt tag

File: content/how-to/account-management/use-role-based-access-control-rbac/index.md [hunk 25]
```diff
@@ -0,0 +1,104 @@
+---
+permalink: use-role-based-access-control-rbac
+audit_date: '2018-06-19'
+title: Use Role Based Access Control (RBAC)
+type: article
+created_date: '2013-06-28'
+created_by: Renee Rendon
+last_modified_date: '2022-11-07'
```
Comment (ssalinas24) [inline content/how-to/account-management/use-role-based-access-control-rbac/index.md:8]: we will update date again

File: content/how-to/account-management/using-RBAC-with-MyRackspace/index.md [hunk 26]
```diff
@@ -0,0 +1,179 @@
+---
+permalink: using-rbac-with-myrackspace
+audit_date: '2022-07-28'
+title: Using RBAC with MyRackspace
+type: article
+created_date: '2022-07-28'
+created_by: Asmita Nakwa
+last_modified_date: '2022-11-07'
```
Comment (ssalinas24) [inline content/how-to/account-management/using-RBAC-with-MyRackspace/index.md:8]: we will update the date again

File: content/how-to/account-management/using-RBAC-with-MyRackspace/index.md (hunk 19 above)
[the remark this answers is not in the document]
Comment (ssalinas24) [reply content/how-to/account-management/using-RBAC-with-MyRackspace/index.md:24]: seems to still be showing. double check.

File: content/how-to/cloud-servers/permissions-matrix-for-role-based-access-control-rbac/index.md [hunk 27]
```diff
@@ -5,8 +5,8 @@ title: Role-Based Access Control (RBAC) permissions matrix for Cloud Hosting
 type: article
 created_date: '2013-06-17'
 created_by: Renee Rendon
-last_modified_date: '2020-10-23'
-last_modified_by: Rose Morales
+last_modified_date: '2022-07-15'
```
Comment (ssalinas24) [inline content/how-to/cloud-servers/permissions-matrix-for-role-based-access-control-rbac/index.md:8]: update the date

File: runtime.txt [hunk 28]
```diff
@@ -1 +1 @@
-3.8
+3.8
```
Comment (ssalinas24) [inline runtime.txt:1]: this change shouldn't be here. This means that some other changes exist in this PR that are not supposed to be here. Please make sure only the changed needed for this article/PR are the ones here.
You likely need to get this PR and Branch updated with master. Master already has the 3.8 change.
- Check your asmita fork repo and make sure the master branch is updated.
- Then on your local machine, `git checkout master`
- don't have any changes pending on master
- then do `git pull` on master branch
- then checkout the branch for this PR, and make sure you don't have any changes pending. If changes pending then commit them.
- then do `git rebase master` while on your current branch.
- then `git push` your branch up to the remote again.
- if it doesn't push then do `git push -f`

File: content/how-to/account-management/using-RBAC-with-MyRackspace/index.md (hunk 19 above)
[the remark this answers is not in the document]
Comment (asmita32675) [reply content/how-to/account-management/using-RBAC-with-MyRackspace/index.md:24]: Done.

File: content/how-to/account-management/overview-role-based-access-control-rbac-cloud/index.md (hunk 22 above)
Comment (asmita32675) [reply content/how-to/account-management/overview-role-based-access-control-rbac-cloud/index.md:98]: Added alt tag

File: content/how-to/account-management/overview-role-based-access-control-rbac-dedicated/index.md (hunk 23 above)
Comment (asmita32675) [reply content/how-to/account-management/overview-role-based-access-control-rbac-dedicated/index.md:94]: added alt tag

File: runtime.txt (hunk 28 above)
Comment (asmita32675) [reply runtime.txt:1]: Done

File: content/how-to/account-management/overview-role-based-access-control-rbac-dedicated/index.md (hunk 24 above)
Comment (asmita32675) [reply content/how-to/account-management/overview-role-based-access-control-rbac-dedicated/index.md]: Added alt tag

File: content/how-to/account-management/overview-role-based-access-control-rbac-dedicated/index.md [hunk 29]
```diff
@@ -0,0 +1,137 @@
+---
+permalink: overview-role-based-access-control-rbac-dedicated
+audit_date: '2022-08-25'
+title: Overview Role Based Access Control for Dedicated
+type: article
+created_date: '2022-08-25'
+created_by: Asmita Nakwa
+last_modified_date: '2022-11-07'
```
Comment (ssalinas24) [inline content/how-to/account-management/overview-role-based-access-control-rbac-dedicated/index.md:8]: we will have to update the date

File: content/how-to/account-management/using-RBAC-with-MyRackspace/index.md [hunk 30]
```diff
@@ -0,0 +1,179 @@
...
+
+**Step 3.** Select the **Group Members** to add and select Save Changes.
+
+**Create Product Group**
+
+**Step 1.** Select Manage Groups
+
+**Step 2.** At the top of the User Product column, enter a name for the new group and select the green plus sign as shown in below image.
+
+<img width="452" alt="myrack-account-RBAC15" src="/support/how-to/using-rbac-with-myrackspace/myrack-account-RBAC15.png">
```
Comment (ssalinas24) [inline content/how-to/account-management/using-RBAC-with-MyRackspace/index.md:133]: alt tag

File: content/how-to/account-management/using-RBAC-with-MyRackspace/index.md [hunk 31]
```diff
@@ -0,0 +1,179 @@
...
+**Delete a User or Product Group**
+Use the following steps to delete a user or product group:
+
+**Step 1.** Select **Manage Groups** from top navigation bar.
+
+**Step 2.** Select an existing group from the User Groups or Product Groups column.
+
+<img width="908" alt="Select an existing user groups" src="/support/how-to/using-rbac-with-myrackspace/myrack-account-RBAC18.png">
+
+**Step 3.** Select the **Actions** dropdown in the upper-right corner as shown in above image.
```
Comment (ssalinas24) [inline content/how-to/account-management/using-RBAC-with-MyRackspace/index.md:163]: are we supposed to be doing `dropdown` or `drop-down`?

File: runtime.txt (hunk 28 above)
Comment (ssalinas24) [reply runtime.txt:1]: Looks like this file change is still coming out. I guess we woul need to find the commit that includes this change, then have to use something like the `git revert` commit, but only if it allowed us to paste the sha for the commit and target only that file and commit.
I guess we will just leave the change in here for now.

**Review by ssalinas24 — CHANGES_REQUESTED** [review]

File: content/how-to/account-management/using-RBAC-with-MyRackspace/index.md (hunk 31 above)
Comment (asmita32675) [reply content/how-to/account-management/using-RBAC-with-MyRackspace/index.md]: Change to drop-down

File: content/how-to/account-management/using-RBAC-with-MyRackspace/index.md (hunk 30 above)
Comment (asmita32675) [reply content/how-to/account-management/using-RBAC-with-MyRackspace/index.md]: Alt tag information added

File: content/how-to/account-management/account-management-faq/index.md [hunk 32]
```diff
@@ -5,17 +5,17 @@ title: Account Management FAQ
 type: article
 created_date: '2019-05-16'
 created_by: Stephanie Fillmon
-last_modified_date: '2020-09-08'
-last_modified_by: Stephanie Fillmon
+last_modified_date: '2020-09-22'
```
Comment (Rohi2305) [inline content/how-to/account-management/account-management-faq/index.md:8]: change the date

File: content/how-to/account-management/account-management-faq/index.md [hunk 33]
```diff
@@ -5,17 +5,17 @@ title: Account Management FAQ
...
+last_modified_by: Asmita Nakwa
 product: Account Management
 product_url: account-management
 ---
 
-### Access control
+### Access Control
 
-{{< accordion title="What is the cost to opt in to RBAC?" col="in" href="accordion16" >}}
+{{< accordion title="What would it cost to choose RBAC?" col="in" href="accordion16" >}}
```
Comment (Rohi2305) [inline content/how-to/account-management/account-management-faq/index.md:16]: keep the old question.

File: content/how-to/account-management/account-management-faq/index.md [hunk 34]
```diff
@@ -28,13 +28,12 @@ Customers can opt out by simply removing the account users.
 Yes, RBAC is available to all Rackspace customers.
 {{</ accordion >}}
 
-{{< accordion title="When I add a user login to an existing contact, can I configure custom role assignments?" col="in" href="accordion19" >}}
+{{< accordion title="When I add a user login credential to an existing contact, can I configure custom role assignments?" col="in" href="accordion19" >}}
```
Comment (Rohi2305) [inline content/how-to/account-management/account-management-faq/index.md:31]:
```suggestion
{{< accordion title="When I add a user login to an existing contact, can I configure custom role assignments?" col="in" href="accordion19" >}}
```

File: content/how-to/account-management/getting-started-with-role-based-access-control-rbac/index.md [hunk 35]
```diff
@@ -5,8 +5,8 @@ title: Getting started with Role-Based Access Control (RBAC)
 type: article
 created_date: '2013-07-15'
 created_by: Renee Rendon
-last_modified_date: '2021-05-06'
-last_modified_by: Carlos Arriaga
+last_modified_date: '2022-08-26'
```
[the remark this answers is not in the document]
Comment (Rohi2305) [reply content/how-to/account-management/getting-started-with-role-based-access-control-rbac/index.md:8]: change the date

File: content/how-to/account-management/overview-role-based-access-control-rbac-cloud/index.md [hunk 36]
```diff
@@ -0,0 +1,161 @@
+---
+permalink: overview-role-based-access-control-rbac-cloud
+audit_date: '2018-02-16'
+title: Overview Role Based Access Control for Cloud
+type: article
+created_date: '2013-04-14'
+created_by: Renee Rendon
+last_modified_date: '2022-11-07'
```
Comment (Rohi2305) [inline content/how-to/account-management/overview-role-based-access-control-rbac-cloud/index.md:8]: change the date

File: content/how-to/account-management/overview-role-based-access-control-rbac-dedicated/index.md (hunk 29 above)
Comment (Rohi2305) [reply content/how-to/account-management/overview-role-based-access-control-rbac-dedicated/index.md:8]: date change

File: content/how-to/account-management/use-role-based-access-control-rbac/index.md (hunk 25 above)
Comment (Rohi2305) [reply content/how-to/account-management/use-role-based-access-control-rbac/index.md:8]: date change

File: content/how-to/account-management/use-role-based-access-control-rbac/index.md [hunk 37]
```diff
@@ -0,0 +1,104 @@
...
+
+**Step 2.** In the upper-right corner of the control panel, click **Account > User Management.**
+
+<img width="946" alt="Select User Management from the drop down menu of Account tab" src="/support/how-to/use-role-based-access-control-rbac/multi_account_rbac1.png">
+
+**Step 3.** On the **User & Permissions** page, click **Create User**.
+
+<img width="945" alt="Click Create user from the page User & Permissions" src="/support/how-to/use-role-based-access-control-rbac/multi_account_rbac2.png">
+
+**Step 4.** Enter information in the **Create a User** section as shown in below image.
```
Comment (Rohi2305) [inline content/how-to/account-management/use-role-based-access-control-rbac/index.md:46]:
```suggestion
**Step 4.** Enter information in the **Create a User** section.
```

File: content/how-to/account-management/using-RBAC-with-MyRackspace/index.md [hunk 38]
```diff
@@ -0,0 +1,179 @@
+---
+permalink: using-rbac-with-myrackspace
+audit_date: '2022-07-28'
+title: Using RBAC with MyRackspace
+type: article
+created_date: '2022-07-28'
+created_by: Asmita Nakwa
+last_modified_date: '2022-11-11'
```
Comment (Rohi2305) [inline content/how-to/account-management/using-RBAC-with-MyRackspace/index.md:8]: change the date

File: content/how-to/account-management/using-RBAC-with-MyRackspace/index.md [hunk 39]
```diff
@@ -0,0 +1,179 @@
...
+
+Ticketing Settings enable you to control the tickets that your users have permission to see when those tickets include one or more devices.
+
+The following table shows the ticketing settings that you need for common actions or views:
+
+{{< table "table  table-striped table-bordered" >}}
+| **Ticket Setting**  | **Functionality** |
+|---------|--------|
+| Flexible Ticket Viewing (default setting) | Enables users to see tickets that include a device that they have permission to view. For example, if a user has access to device A, they see all of the tickets that include device A, even if the ticket has other devices. If device C is later added to the ticket, the user can still see the ticket, even if the user does not have access to device C. |
+| Strict Ticket Viewing | Requires users to have access to all the devices on a ticket to see that ticket. For example, if a ticket includes device A and device B, the user must have permissions to both devices to see the ticket. If a ticket the user could see later includes device C, to which the user has no access, the user can no longer see the ticket. |
```
Comment (Rohi2305) [inline content/how-to/account-management/using-RBAC-with-MyRackspace/index.md:179]: check the below images.

File: content/how-to/cloud-servers/permissions-matrix-for-role-based-access-control-rbac/index.md (hunk 27 above)
Comment (Rohi2305) [reply content/how-to/cloud-servers/permissions-matrix-for-role-based-access-control-rbac/index.md:8]: date change

**Review by Rohi2305 — COMMENTED** [review]
Update all the comments

File: content/how-to/account-management/overview-role-based-access-control-rbac-dedicated/index.md (hunk 29 above)
Comment (asmita32675) [reply content/how-to/account-management/overview-role-based-access-control-rbac-dedicated/index.md:8]: Done

File: content/how-to/account-management/account-management-faq/index.md (hunk 32 above)
Comment (asmita32675) [reply content/how-to/account-management/account-management-faq/index.md:8]: Done

File: content/how-to/account-management/account-management-faq/index.md (hunk 33 above)
Comment (asmita32675) [reply content/how-to/account-management/account-management-faq/index.md:16]: Done

File: content/how-to/account-management/account-management-faq/index.md (hunk 34 above)
Comment (asmita32675) [reply content/how-to/account-management/account-management-faq/index.md:31]: Done

File: content/how-to/account-management/overview-role-based-access-control-rbac-cloud/index.md (hunk 36 above)
Comment (asmita32675) [reply content/how-to/account-management/overview-role-based-access-control-rbac-cloud/index.md:8]: Done

File: content/how-to/account-management/use-role-based-access-control-rbac/index.md (hunk 37 above)
Comment (asmita32675) [reply content/how-to/account-management/use-role-based-access-control-rbac/index.md:46]: Done

File: content/how-to/account-management/using-RBAC-with-MyRackspace/index.md (hunk 38 above)
Comment (asmita32675) [reply content/how-to/account-management/using-RBAC-with-MyRackspace/index.md:8]: Done

**Review by Rohi2305 — APPROVED** [review]

**Review by ssalinas24 — APPROVED** [review]

**Pull request merged**